pytorch_lightning/loops/base.py

```diff
@@ -275,10 +276,9 @@ def state_dict(self, destination: Optional[Dict] = None, prefix: str = "") -> Di
         destination[prefix + "state_dict"] = self.on_save_checkpoint()
 
         # do not get the mode from `self.trainer` because it might not have been attached yet
-        ft_enabled = _FaultTolerantMode.detect_current_mode().is_enabled
         for k, v in self.__dict__.items():
             key = prefix + k
-            if ft_enabled and isinstance(v, BaseProgress):
+            if isinstance(v, BaseProgress):
```

Comment:
Why did you modify this?

Reply:
progress state is saved only when fault_tolerant is enabled, but now with the tuner, we need to reset the base progress as well after tuning if it's not early exiting to make sure while continuing with the `.fit/validate/..` call it should be reset completely.